static/app/views/issueList/utils.tsx

```diff
@@ -1,12 +1,13 @@
 import type {Location, LocationDescriptorObject} from 'history';
 
 import ExternalLink from 'sentry/components/links/externalLink';
-import {DEFAULT_QUERY} from 'sentry/constants';
 import {t, tct} from 'sentry/locale';
 import type {Event} from 'sentry/types/event';
 import type {Group, GroupTombstoneHelper} from 'sentry/types/group';
 import type {Organization} from 'sentry/types/organization';
 
+export const DEFAULT_QUERY = 'is:unresolved issue.priority:[high, medium]';
+
 export enum Query {
   FOR_REVIEW = 'is:unresolved is:for_review assigned_or_suggested:[me, my_teams, none]',
   PRIORITIZED = DEFAULT_QUERY, // eslint-disable-line @typescript-eslint/prefer-literal-enum-member
```

Comment:
note: verbatimModuleSyntax also enables [isolatedModules](https://www.typescriptlang.org/tsconfig/#isolatedModules), where files must be parseable on their own. For enums, that means they can’t reference a value outside of its file, or it’ll error with:

```
TS18055: 'Query.PRIORITIZED' has a string type, but must have syntactically recognizable string syntax when 'isolatedModules' is enabled.
```

This is the only place where we had this violation (it’s also a lint violation as we can see), and since `DEFAULT_QUERY` is only used within `views/issueList`, it makes sense to inline it here, as this also makes the constant smaller in scope / less global.